src/atlantis/supervisor/rpc/rpc_test.go

```diff
@@ -132,6 +133,8 @@ func (s *RpcSuite) TestTeardown(c *gocheck.C) {
 	reply = SupervisorTeardownReply{}
 	c.Assert(ih.Teardown(arg, &reply), gocheck.IsNil)
 	c.Assert(reply.Status, gocheck.Equals, StatusOk) // gocheck doesn't have a NotEquals?!
+	// sort container ids before Assert
+	sort.Strings(reply.ContainerIDs)
 	c.Assert(reply.ContainerIDs, gocheck.DeepEquals, []string{"theContainerID3", "theContainerID4",
```

Comment:
Assign `[]string{"theContainerID3", "theContainerID4","theContainerID5"}` to variable, and then sort it. When new string is added, developer need not worry about manually sorting the string array.